tests/test_dse.py

```diff
@@ -74,44 +74,53 @@ def test_scheduling_after_rewrite():
     # Dimension-independent data dependences should be a stopper for CSE
     (['Eq(tu.forward, tu.dx + 1)', 'Eq(tv.forward, tv.dx + 1)',
       'Eq(tw.forward, tv.dt + 1)', 'Eq(tz.forward, tv.dt + 2)'],
-     ['1/h_x', '1/dt', '-r1*tv[t, x, y, z]',
-      '-r2*tu[t, x, y, z] + r2*tu[t, x + 1, y, z] + 1',
-      '-r2*tv[t, x, y, z] + r2*tv[t, x + 1, y, z] + 1',
-      'r0 + r1*tv[t + 1, x, y, z] + 1',
-      'r0 + r1*tv[t + 1, x, y, z] + 2']),
+     ['1/h_x', '-r1*tu[t, x, y, z] + r1*tu[t, x + 1, y, z] + 1',
```

Comment:
Note: This is due to the new "dependent" insertion rule for the temporary that changes slightly order of things but it is still correct.